packages/blob/package.json

```diff
@@ -11,26 +11,16 @@
   "license": "Apache-2.0",
   "sideEffects": false,
   "type": "module",
-  "exports": {
```

Comment:
Are you 100% sure this will generate a package that will work when imported in:
- when used in a regular node.js script
- as a nextjs application dependency
- an edge function in nextjs
- a browser

For some reason, I remember having spent a good amount of time to find we had to put these lines. If you manage to check it's working without them in the various contexts fine 👍

Reply:
I created e2e tests for edge, node and browser. I also run the script 
```
const blob = require('@vercel/blob');

blob
  .list({
    token: '...',
  })
  .then((r) => console.log(r));

```
inside `test/next`

Reply:
👍

Reply:
@correttojs if these lines are not needed for the module to work in various situations then yes we can remove them 👍